### PR TITLE
update to v0.2.0 release

### PR DESCRIPTION
**Pull request by phobson**

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by phobson** [comment]
@ocefpaf i didn't know we could as the bot to do things for us!

**Comment by ocefpaf** [comment]
> @ocefpaf i didn't know we could as the bot to do things for us!

There are many things we can do with the bots now:

https://conda-forge.org/docs/webservice.html

There is also the auto-updating bot. When you issue a new release on PyPI or GitHub you can wait a few hours and the bot will trigger a PR for you.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,13 +17,13 @@ build:
 requirements:
   build:
     - python
+    - pip
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Sorry, forgot that in my edits.

**Comment by phobson** [comment]
@ocefpaf I'm getting gunshy about mashing the green button here (goes against the convention the MPL devs have that says the PR opener should not be the merger). Think this is ready?

File: recipe/meta.yaml (hunk 1 above)
Comment (phobson) [reply recipe/meta.yaml]: no problem -- I should have noticed

**Comment by ocefpaf** [comment]
> @ocefpaf I'm getting gunshy about mashing the green button here (goes against the convention the MPL devs have that says the PR opener should not be the merger).

For conda-forge feedstocks it is quite the opposite. We encourage self-merges, specially for version bumps. However, when in doubt feel free to ping me.

> Think this is ready?

Yep.

**Pull request merged**